lib-utilities/config/odimra_config.json

```diff
@@ -58,6 +58,13 @@
 	},
 	"AddComputeSkipResources": { 
 		"SystemCollection": [ 
+			"Chassis",
+			"Managers",
+			"LogServices",
```

Comment:
I guess these URIs are present in the systems json . So what happens when a user queries for these URI ?

Reply:
User will be able to access all the URI's. This list is for updating data into DB, while adding system details, manager,chassis and logservices are ignored as there is a separate process for it already